screens/ChatScreen.js

```diff
@@ -111,6 +113,17 @@ export default class ChatScreen extends React.Component {
     }
 }
 
+function mapStateToProps(state) {
+    return {
+      //chatRooms: state.chatRooms
+      messages: state.messages
+      //userID: state.userID
+    };
+}
+
+const mapDispatchToProps = {
+  };
```

Comment:
just to put an example in here it'd be something like `const mapDispatchToProps = dispatch => ({getMessages: bindActionCreators(getMessages, dispatch)});`

Reply:
Yeah I read about using bindActionCreators like that, seemed nicer to just lay it out like in ChatsScreen though I'm not super familiar with the differences:

```
const mapDispatchToProps = {
    createMessage,
    setCurrentUser,
    setCurrentChat,
    createChat
  };
```

Reply:
This is also discussed in the [documentation](https://redux.js.org/api/bindactioncreators).